vj4/test/test_model.py

```diff
@@ -21,13 +24,15 @@
 DUP_UNAME = 'GuESt'
 UID = 22
 UNAME = 'twd2'
-OWNER_UID = 22
+OWNER_UID = -1
```

Comment:
???

Reply:
There are tests that adding user=UID into a domain owned by user=OWNER_UID, so that UID and OWNER_UID should be different for that test to run.